### PR TITLE
Docs update for ns-date-time + ns-alert (dismissed event)

### PR DESCRIPTION
**Pull request by PtitBen**

### CODE REVIEW

File: docs/components/ns-date-time.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Component placement
+
+ns-date-time can be used in any html text element.
+
+## Specification
+
+| Attribute | Property | Type | Default | Options | Description |
+| :--- | :--- | :--- | :--- | :--- |-------------|
+| `date` | `date` | `string` | `YYYY-MM-DDTHH:MM, YYYY-MM-DD, HH:MM` | | The date and/or time to display |
```
Comment (andij) [inline docs/components/ns-date-time.md:38]: Should these be separated?

```suggestion
| `date` | `date` | `string` | `YYYY-MM-DDTHH:MM`, `YYYY-MM-DD`, `HH:MM` | | The date and/or time to display |
```

File: docs/components/ns-date-time.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| `hideDate` | `hideDate` | `boolean` | `false` | | Whether to hide the date |
```
Comment (andij) [inline docs/components/ns-date-time.md:39]: Attribute is kebab-case.
```suggestion
| `hide-date` | `hideDate` | `boolean` | `false` | | Whether to hide the date |
```

File: docs/components/ns-date-time.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+| `hideTime` | `hideTime` | `boolean` | `false` | | Whether to hide the time |
```
Comment (andij) [inline docs/components/ns-date-time.md:40]: Attribute is kebab-case.
```suggestion
| `hide-time` | `hideTime` | `boolean` | `false` | | Whether to hide the time |
```

File: docs/components/ns-date-time.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+| `hideDay` | `hideDay` | `boolean` | `false` | | Whether to hide the day |
```
Comment (andij) [inline docs/components/ns-date-time.md:41]: Attribute is kebab-case.
```suggestion
| `hide-day` | `hideDay` | `boolean` | `false` | | Whether to hide the day |
```

File: docs/components/ns-date-time.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+| `hideMonth` | `hideMonth` | `boolean` | `false` | | Whether to hide the month |
```
Comment (andij) [inline docs/components/ns-date-time.md:42]: Attribute is kebab-case.
```suggestion
| `hide-month` | `hideMonth` | `boolean` | `false` | | Whether to hide the month |
```

File: docs/components/ns-date-time.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+| `hideYear` | `hideYear` | `boolean` | `false` | | Whether to hide the year |
```
Comment (andij) [inline docs/components/ns-date-time.md:43]: Attribute is kebab-case.
```suggestion
| `hide-year` | `hideYear` | `boolean` | `false` | | Whether to hide the year |
```

File: docs/components/ns-date-time.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+<StorybookStory story="components-ns-date-time--standard"></StorybookStory>
+
+## Component placement
+
+ns-date-time can be used in any html text element.
+
+## Specification
+
+| Attribute | Property | Type | Default | Options | Description |
```
Comment (andij) [inline docs/components/ns-date-time.md:36]: Consider removing the Options column as there's no content.

Comment (PtitBen) [reply docs/components/ns-date-time.md:36]: I think you spotted a problem, actually ...
The options should be the different strings that I have mistakenly put in the default column.

**Pull request merged**